### ISSUE TITLE
Cambio de icono

### ISSUE THREAD

**Issue by adriana-rv** [imatem/imtheme.blue#28]
Al abrir la  página del IM, en el navegador se muestra el icono del plone y no del IM

**Comment by gil-cano** [comment]
El icono solo de esta mostrando en la página principal, el las otras páginas se muestra el icono de plone. 
Se deben revisar las reglas que copian el JavaScript y CSS al header.

**Comment by adriana-rv** [comment]
No, si se selecciona otra sección el icono continua siendo el del plone, ejemplo de ello son:
https://www.matem.unam.mx/fsd
https://www.matem.unam.mx/acerca-de
